symfony/src/Controller/CustomerController.php

```diff
@@ -58,7 +58,15 @@ public function get($id)
             return new Response($errorsString);
         }
         $jsonContent = $this->serializer->serialize($customer, 'json');
-        return new Response($jsonContent, Response::HTTP_OK);
+        $dto = $this->serializer->deserialize($jsonContent, CustomerDTO::class, 'json');
+        //validate fields
+        $errors = $this->validator->validate($dto);
+        if (0 === count($errors)) {
+            return new Response($jsonContent, Response::HTTP_OK);
+        }else{
+            $jsonContent = $this->serializer->serialize($errors,'json');
+            return new Response($jsonContent, Response::HTTP_OK);
```

Comment:
Response with errors can't have response code `Response::HTTP_OK` (200 OK). It should be `Response:: HTTP_BAD_REQUEST` (400 Bad Request). Take a look into available response codes: https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/HttpFoundation/Response.php#L23